dallinger/command_line.py

```diff
@@ -106,13 +106,14 @@ def setup_experiment(debug=True, verbose=False, app=None, exp_config=None):
     # Verify that the Postgres server is running.
     try:
         psycopg2.connect(database="x", user="postgres", password="nada")
-    except psycopg2.OperationalError, e:
+    except psycopg2.OperationalError as e:
         if "could not connect to server" in str(e):
             raise RuntimeError("The Postgres server isn't running.")
 
-    # Load psiTurk configuration.
-    psi_config = PsiturkConfig()
-    psi_config.load_config()
+    # Load configuration.
+    config = get_config()
+    if not config.ready:
+        config.load_config()
```

Comment:
Indeed this was an oversight during the merging of config changes. I've addressed this on a new branch that implements the new configuration object more widely (https://github.com/Dallinger/Dallinger/pull/389).